File: CS/E911.Module.Web/Controllers/WebShowCustomFormWindowController.cs
using System;
using E911.Module.Controllers;

namespace E911.Module.Web.Controllers {
    public class WebShowCustomFormWindowController : ShowCustomFormWindowController {
        protected override void ShowCustomForm() {
            DevExpress.ExpressApp.Web.WebWindow.CurrentRequestWindow.RegisterStartupScript(
                "ShowCustomFormScript", "window.open('Controls/WebCustomForm.aspx');");
        }
    }
}

File: VB/E911.Module.Web/Controllers/WebShowCustomFormWindowController.vb
Imports Microsoft.VisualBasic
Imports System
Imports E911.Module.Controllers

Namespace E911.Module.Web.Controllers
	Public Class WebShowCustomFormWindowController
		Inherits ShowCustomFormWindowController
		Protected Overrides Sub ShowCustomForm()
			DevExpress.ExpressApp.Web.WebWindow.CurrentRequestWindow.RegisterStartupScript("ShowCustomFormScript", "window.open('Controls/WebCustomForm.aspx');")
		End Sub
	End Class
End Namespace